lib/svg.dart

```diff
@@ -617,11 +617,7 @@ class _SvgPictureState extends State<SvgPicture> {
     }
 
     if (_picture != null) {
-      final double devicePixelRatio =
-          MediaQuery.of(context, nullOk: true)?.devicePixelRatio ??
-              window.devicePixelRatio;
-      final Rect viewport =
-          Offset.zero & (_picture.viewport.size * devicePixelRatio);
+      final Rect viewport = Offset.zero & _picture.viewport.size;
```

Comment:
You'll need to remov the `, window` part of the `dart:ui` import now to make analysis happy.

Reply:
Fixed. I did not realise the import warning was a result of my change.